repologyapp/templates/admin/cpes.html

```diff
@@ -56,7 +56,7 @@
 	{% for cpe in cpes %}
 		<tr>
 			<td class="minimal-column"><a href="{{ url_for('project_versions', name=cpe.effname) }}">{{ cpe.effname }}</a></td>
-			<td class="minimal-column">{{ mcpes.panels(cpe) }}</td>
+			<td class="minimal-column">{{ panels(cpe) }}</td>
```

Comment:
`panels` is too ambiguous. Please import `panels as cpe_panels` or `import "_macros/cpes.html as cpes` and invoke as `cpe_panels` or `cpe.panels`.

Reply:
I agree that it's ambiguous. But maybe it's better to rename ambiguous entries instead? That way we can consistently use `from _macros/XXX import YYY` everywhere.

What do you think about these names?

`meta` -> `enable_autorefresh`
`toggler` -> `autorefresh_toggler`
`panels` -> `cpe_components`